### PR TITLE
Add End-History

### PR DESCRIPTION
**Pull request by JoseDRojasA**

### CODE REVIEW

File: frontend/src/app/main/components/end-history-wrapper/end-history-wrapper.component.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { MatDialog } from '@angular/material/dialog';
+import { EndHistoryComponent } from '../../components/end-history/end-history.component';
+
+@Component({
+  selector: 'aequi-end-history-wrapper',
+  templateUrl: './end-history-wrapper.component.html',
+  styleUrls: ['./end-history-wrapper.component.scss']
+})
+export class EndHistoryWrapperComponent implements OnInit {
+
```
Comment (JuanDiegoLopez) [inline frontend/src/app/main/components/end-history-wrapper/end-history-wrapper.component.ts]: You can remove this empty line

File: frontend/src/app/main/components/end-history-wrapper/end-history-wrapper.component.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    this.dialog.open(EndHistoryComponent, {
+      data: {
+        emoji: this.emoji,
+        color: this.color,
+        message: this.message
+      },
+      panelClass: 'end-story'
+    })
+  }
+
```
Comment (JuanDiegoLopez) [inline frontend/src/app/main/components/end-history-wrapper/end-history-wrapper.component.ts:27]: You can remove empty line

File: frontend/src/app/main/components/end-history-wrapper/end-history-wrapper.component.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  ngOnInit(): void {
+    this.dialog.open(EndHistoryComponent, {
+      data: {
+        emoji: this.emoji,
+        color: this.color,
+        message: this.message
+      },
+      panelClass: 'end-story'
+    })
```
Comment (JuanDiegoLopez) [inline frontend/src/app/main/components/end-history-wrapper/end-history-wrapper.component.ts]: add missing semicolon

File: frontend/src/app/main/components/end-history/end-history.component.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (JuanDiegoLopez) [inline frontend/src/app/main/components/end-history/end-history.component.html]: Remove empty line

File: frontend/src/app/main/components/end-history/end-history.component.scss [hunk 5]
```diff
@@ -0,0 +1,26 @@
+main {
+  position: relative;
+  padding: 50px;
+  text-align: center;
+  display: grid;
+  height: 70vh;
+  label.close {
```
Comment (JuanDiegoLopez) [inline frontend/src/app/main/components/end-history/end-history.component.scss:8]: please add break line after each class definition

**Review by JuanDiegoLopez — APPROVED** [review]
Approved!

File: frontend/src/app/main/components/end-history/end-history.component.html (hunk 4 above)
Comment (JoseDRojasA) [reply frontend/src/app/main/components/end-history/end-history.component.html:1]: It's not possible :(

**Pull request merged**